fuzz/Cargo.toml

```diff
@@ -12,11 +12,7 @@ cargo-fuzz = true
 libfuzzer-sys = "0.4"
 
 [dependencies.openmls]
-path = ".."
-
-# Prevent this from interfering with workspaces
-[workspace]
-members = ["."]
+path = "../openmls"
```

Comment:
Pulling the fuzzing crate into the workspace isn't great.
With this we are benchmarking the fuzzers when running `cargo bench` in the root directory.  How do you think this should be used?

Reply:
Nice catch, I've disable `cargo bench` lookup of the fuzzer binaries manually with the appropriate flags (`bench = false` & `harness = false`)